app/subscriber/src/components/sidebar/advanced-search/AdvancedSearch.tsx

```diff
@@ -87,8 +88,9 @@ export const AdvancedSearch: React.FC<IAdvancedSearchProps> = ({ expanded, setEx
           Object.values(advancedSearch.searchInField).every((v) => v === false)
             ? advancedSearch.searchTerm
             : '',
+        productIds: advancedSearch?.frontPage ? [11] : [],
```

Comment:
Hardcoding primary keys will almost always lead to problems.  Extract them to an environmental config.